src/ic-cdk/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [unreleased]
+## [0.5.1] - 2022-05-16
```

Comment:
I think the normal practice is to leave the `## [unreleased]` section at the top, as a place for new entries to go
```suggestion
## [unreleased]

## [0.5.1] - 2022-05-16
```